mono/mini/Makefile.am.in

```diff
@@ -244,7 +244,7 @@ LLVMMONOF=$(LLVM_LIBS) $(LLVM_LDFLAGS)
 endif
 
 if !HOST_WIN32
-if HAVE_ZLIB
+if HAVE_SYS_ZLIB
```

Comment:
do you know why mini/ has this zlib logic here? It doesn't use any zlib pieces from what I can see. Can you try removing this?